test/integrations/destinations/pagerduty/router/data.ts

```diff
@@ -267,5 +267,11 @@ export const data = [
         },
       },
     },
+    mockFns: (_) => {
+      jest.spyOn(Date, 'now').mockImplementation(() => {
+        console.log('Pagerduty router mock');
```

Comment:
Is the log helpful here?  Why are not using it in other test cases 

```suggestion
```

Reply:
Done